airflow/models/__init__.py

```diff
@@ -4012,17 +4088,55 @@ def create_dagrun(self,
             conf=conf,
             state=state
         )
+
+        dag_model = DagModel.get_dagmodel(self.dag_id)
+
+        tis = self.create_tis(execution_date)
+        edges = self.create_edges(-1)
```

Comment:
what does -1 mean?

Reply:
Ye this is a bit magic so I did make it now a name argument and adding a comment why I do this. This is the graph_id, -1 is just a temp value, will be set in a later stage